samples/kotlin-2-sample/shared/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
 plugins {
     alias(libs.plugins.kotlinMultiplatform)
     alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.compose.compiler) apply false
```

Comment:
if we not use it here - why it was added?

Reply:
my mistake, incorrect understand warning for new compiler settings, removed